packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1424,6 +1430,14 @@ class ThemeData with Diagnosticable {
   ButtonBarThemeData get buttonBarTheme => _buttonBarTheme!;
   final ButtonBarThemeData? _buttonBarTheme;
 
+  /// The background color of [Dialog] elements.
+  @Deprecated(
+    'Use DialogThemeData.backgroundColor instead. '
+    'This feature was deprecated after v3.27.0-0.1.pre.',
+  )
+  Color? get dialogBackgroundColor => _dialogBackgroundColor;
```

Comment:
Since this is made nullable where it previously was not, customer code in the google testing shard will not compile where is was previously Color, and now is Color? and in need of a null check.

Reply:
Ahh...Thanks! Usually we made the deprecated property nullable in `ThemeData.raw` . I did same for button bar just above this.  Just the updated the PR to revert this to be non-nullable.

Reply:
Perfect thank you! There were over a thousand failures. If there were less I would have tried to migrate the customers, but this one clearly has a big impact. :)